fc/jira/tasks.py

```diff
@@ -7,7 +7,7 @@
 from .triage_task import TriageTask
 
 
-search_url = 'https://jira.cms.gov/rest/api/2/search?jql={}'
+search_url = 'https://jira.cms.gov/rest/api/2/search?maxResults=100&jql={}'
```

Comment:
We only want to find, and score, up to 100 tasks?  What if one of the tasks that would be skipped was supposed to have a high VFR?